public/app/plugins/datasource/jaeger/_importedDependencies/types/config.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
Where is any exported member in this file used? It's also in the /TraceView/ but not sure where it's used.